tests/contrib/logbook/test_logbook_logging.py

```diff
@@ -18,6 +18,9 @@
 def _test_logging(span, env, service, version):
     dd_trace_id, dd_span_id = (span.trace_id, span.span_id) if span else (0, 0)
 
+    if dd_trace_id != 0 and config._128_bit_trace_id_enabled and not config._128_bit_trace_id_logging_enabled:
+        dd_trace_id = span._trace_id_64bits
```

Comment:
nit: I dont think we need to check if `128_bit_trace_id` is enabled. 

`span._trace_id_64bits == trace_id` if `config._128_bit_trace_id_enabled == False`.

Reply:
Done